src/policy/CMSRucioPolicy/algorithms/auto_approve.py

```diff
@@ -63,14 +74,19 @@ def global_approval(did, rule_attributes, session):
     except (NoOptionError, NoSectionError, RuntimeError):
         RULE_LIFETIME_THRESHOLD = 2592000
 
-    account = rule_attributes['account']
+    try:
+        SINGLE_RSE_RULE_SIZE_THRESHOLD = float(config_get('rules', 'single_rse_rule_size_threshold', raise_exception=True, default=50e12))
+    except (NoOptionError, NoSectionError, RuntimeError):
+        SINGLE_RSE_RULE_SIZE_THRESHOLD = 50e12
+
+    AUTO_APPROVE_ACTIVITY = 'User AutoApprove'
```

Comment:
This (and everything else in all caps) is not a program level constant. So please don't use all caps.